README.md

```diff
@@ -2,61 +2,60 @@
 
 [![CircleCI](https://circleci.com/gh/idpass/idpass-lite-java.svg?style=svg&circle-token=4fb5cc4cfe96b754d1842c2443ee638608bc4755)](https://circleci.com/gh/idpass/idpass-lite-java)
 
-This is a Java wrapper of the [libidpasslite](https://github.com/idpass/idpass-lite) library that provides developers with an API to create and interact with ID PASS Lite cards.
+A Java wrapper for the [idpass-lite](https://github.com/idpass/idpass-lite) library, providing an API to create and interact with secure and biometrically-binding QR code identity cards.
 
 ![id front](testdata/idpass-lite-java-sample-front.png?raw=true "front") ![id back](testdata/idpass-lite-java-sample-back.png?raw=true "back")
 
-
-## Building
-```bash
-./gradlew build
-```
-
 ## Features
-- Create card with face
-- Verify card with face
-- Verify card with pin
-- Sign with card
-- Encrypt with card
-- Add/revoke/verify certificates
-- Generate QR Code
-- Read QR Code
 
-## Quick start
-This library is used to generate a secure and biometrically-binding QR coded identification cards. 
+- Create and verify card with face
+- Verify card with PIN
+- Sign and encrypt with card
+- Add, revoke, and verify certificates
+- Generate and read QR codes
 
-### 1. Install
-Declare Maven Central repository in the dependency configuration. For example, in `build.gradle`:
+## Installation
+
+Declare Maven Central repository in the dependency configuration, then add this library in the dependencies. An example using `build.gradle`:
 
 ```groovy
 repositories {
-    mavenCentral()
+  mavenCentral()
 }
 
-dependencies {
-    implementation "org.idpass:idpass-lite-java:0.0.1-SNAPSHOT"
-    implementation 'com.google.protobuf:protobuf-java:3.12.2'
+implementation "org.idpass:idpass-lite-java:0.0.1-SNAPSHOT"
+implementation "com.google.protobuf:protobuf-java:3.12.2"
```

Comment:
Why removing the indentation?

Reply:
Oops this was a mistake, already fixed in latest commit. Will coordinate with @typelogic to add the sample code in the README into the test suite as well.

Reply:
Sample code now added to the test suite, thanks @typelogic ! 👍